src/main/java/Bao.java

```diff
@@ -0,0 +1,116 @@
+import Components.Task;
+import Components.Todo;
+import Components.Deadline;
+import Components.Event;
+
+import java.util.Scanner;
+
+public class Bao {
+    private static Scanner in = new Scanner(System.in);
+    private static Task[] tasks = new Task[100];
+    private static int numTasks=0;
+
+    private static void greet(){
+        System.out.println("______________________________________________________________________________________");
+        System.out.println("Hello there! Bao here!");
+        System.out.println("How can I help?");
+        System.out.println("______________________________________________________________________________________");
+    }
+
+    private static void addTask(Task task){
+        System.out.println("______________________________________________________________________________________");
+        tasks[numTasks++]=task;
```

Comment:
I think it would be better to have a space separating the equals sign like you did in other lines.
```suggestion
        tasks[numTasks++] = task;
```